### PR TITLE
fix: manually configure purgecss

### PR DESCRIPTION
**Pull request by janvennemann**

### CODE REVIEW

File: docs/.vuepress/plugins/purgecss.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+   */
+  const whitelistPatterns = [
+    /^(h\d|p$|ul|li$|div|ol|table|td$|th$|thead|tbody|main|input|button|form|md-|hljs)/
+  ]
+  const purgecss = require('@fullhuman/postcss-purgecss')
+  siteConfig.postcss.plugins.push(purgecss({
+    content: [
+      `${sourceDir}/**/*.md`,
+      `${sourceDir}/**/*.vue`,
+      `${sizrceDir}/../website/pages/**/*.md`,
```
Comment (sgtcoolguy) [inline docs/.vuepress/plugins/purgecss.js]: I assume these are typos?
```suggestion
      `${sourceDir}/../website/pages/**/*.md`,
```

File: docs/.vuepress/plugins/purgecss.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      `${sizrceDir}/../website/pages/**/*.vue`
```
Comment (sgtcoolguy) [inline docs/.vuepress/plugins/purgecss.js]:
```suggestion
      `${sourceDir}/../website/pages/**/*.vue`
```

File: docs/.vuepress/plugins/purgecss.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    ],
+    whitelistPatterns,
+    defaultExtractor: context => {
```
Comment (sgtcoolguy) [inline docs/.vuepress/plugins/purgecss.js]:
```suggestion
    defaultExtractor: content => {
```

**Review by sgtcoolguy — CHANGES_REQUESTED** [review]

**Comment by sgtcoolguy** [comment]
I commented out the guard to not run purges on dev builds and tried this still results in a mangled UI with seemingly no CSS.

**Comment by janvennemann** [comment]
@sgtcoolguy i updated the globs to include all VuePress plugins and our theme, it should keep all used CSS now

**Comment by sgtcoolguy** [comment]
Well, we're closer - but something still seems funky. I tried this locally rebased on master, and the external link markers get... interesting:
Screen Shot 2020-10-20 at 11 16 36 AM
Screen Shot 2020-10-20 at 11 16 44 AM

**Comment by sgtcoolguy** [comment]
ok, figured it out, needed to include `node_modules/@vuepress/core/lib/**/*.vue`

**Review by sgtcoolguy — APPROVED** [review]

**Pull request merged**